### PR TITLE
add todolist code

### PR DESCRIPTION
**Pull request by lorenzoworx**
Add code from todo list project for peer review to ensure best practices are used

### CODE REVIEW

File: src/modules/crud.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import Storage from './storage.js';
+import { taskList, renderList } from './render.js';
+import Task from './task.js';
+
+export default class List {
+  constructor() {
+    this.myTasks = [];
+    this.store = new Storage();
```
Comment (ikoote1) [inline src/modules/crud.js:8]: The site works very well with a perfect description and readme file.
_But there are some little things which can make it better._

One of the few things which need some checking is, the declaration of an object inside a constructor.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 10px;
+}
+.borderBottom { border-bottom: 1px solid #ccc; }
```
Comment (ikoote1) [inline src/style.css:30]: I really appreciate the way you style your project, the output is very nice. but If it's ok with you. I would suggest that you use some contest properties for better CSS code. 

Thanks for the nice work.

**Review by ikoote1 — CHANGES_REQUESTED** [review]
Overall, your work is nice and everything is almost perfect. 

- The functionality of the List is ok.
- Nice description.
- Wonderful readme file
- No linter errors

**Review by ikoote1 — APPROVED** [review]
project Approved.

Thanks for working on all the requested changes. Now you are good to go.

File: src/style.css (hunk 2 above)
Comment (lorenzoworx) [reply src/style.css:30]: Duly noted

**Pull request merged**